src/pdn/src/PdnGen.tcl

```diff
@@ -6238,13 +6263,16 @@ proc generate_voltage_domain_rings {core_ring_data} {
             [expr $outer_ux + $width / 2] \
             [expr $outer_uy + $width / 2] \
           ]
+        
+        #get the name of inner power(first power) of voltage domain
+        set first_domain_power [lindex $power_net 0]
 
-        if {$power_net == [get_voltage_domain_power [dict get $design_data core_domain]]} {
+        if {$first_domain_power == [get_voltage_domain_power [dict get $design_data core_domain]]} {
```

Comment:
I'm not convinced that we should treat CORE power/ground differently - why cant all stripes be added with the tag POWER_<net_name> or GROUND_<net_name>?

Reply:
Similar to this. https://github.com/The-OpenROAD-Project/OpenROAD/pull/1118#discussion_r716957405